core/src/main/java/com/backblaze/b2/client/B2LargeFileUploader.java

```diff
@@ -225,23 +226,25 @@ private B2FileVersion uploadPartsAndFinish(B2FileVersion largeFileVersion,
         B2FinishLargeFileRequest finishRequest = B2FinishLargeFileRequest
                 .builder(largeFileVersion.getFileId(), partSha1s)
                 .build();
-        return backoffRetryer.doRetry(accountAuthCache, () -> webifier.finishLargeFile(accountAuthCache.get(), finishRequest));
+        return retryer.doRetry(accountAuthCache, () -> webifier.finishLargeFile(accountAuthCache.get(), finishRequest), new B2DefaultRetryPolicy());
     }
 
     private B2Part uploadOnePart(B2UploadPartUrlCache uploadPartUrlCache,
                                  B2UploadFileRequest request,
                                  B2PartSpec partSpec) throws B2Exception {
-        return backoffRetryer.doRetry(accountAuthCache, (isRetry) -> {
-            final B2UploadPartUrlResponse uploadPartUrlResponse = uploadPartUrlCache.get(isRetry);
-
-            final B2UploadPartRequest partRequest = B2UploadPartRequest
-                    .builder(partSpec.partNumber,
-                            new B2PartOfContentSource(request.getContentSource(), partSpec.start, partSpec.length))
-                    .build();
-
-            final B2Part part = webifier.uploadPart(uploadPartUrlResponse, partRequest);
-            uploadPartUrlCache.unget(uploadPartUrlResponse);
-            return part;
-        });
+        return retryer.doRetry(accountAuthCache,
+                (isRetry) -> {
+                    final B2UploadPartUrlResponse uploadPartUrlResponse = uploadPartUrlCache.get(isRetry);
+
+                    final B2UploadPartRequest partRequest = B2UploadPartRequest
+                            .builder(partSpec.partNumber,
+                                    new B2PartOfContentSource(request.getContentSource(), partSpec.start, partSpec.length))
+                            .build();
+
+                    final B2Part part = webifier.uploadPart(uploadPartUrlResponse, partRequest);
+                    uploadPartUrlCache.unget(uploadPartUrlResponse);
+                    return part;
+                },
+                new B2DefaultRetryPolicy());
```

Comment:
yes.  good eye.  :)

right now, we always use the default retryer.  in an upcoming change, i'll make it configurable from the outside and i'll fix it here too.